### PR TITLE
Add Press TV French and update logos

### PR DESCRIPTION
**Pull request by UltraHDR**

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
https://github.com/iptv-org/database/actions/runs/3353282994/jobs/5555967342

```
18639  "logo" has an invalid file extension ".gif"
```

We only keep links to PNG or JPEG versions of logo: https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

And what's wrong with the old logo?

File: data/channels.csv [hunk 1]
```diff
@@ -18636,7 +18636,8 @@ PremiumStories.it,Premium Stories,,,,IT,,,c/IT,ita,,FALSE,2015-06-23,2022-01-10,
 PremiumStoriesHD.it,Premium Stories HD,,,,IT,,,c/IT,ita,,FALSE,2018-06-04,2022-01-10,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PremiumStoriesPlus24.it,Premium Stories +24,,,,IT,,,c/IT,ita,,FALSE,2016-03-01,2017-09-04,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PresenceTVChannel.et,Presence TV Channel,,,,ET,,,c/ET,amh,,FALSE,,,,,https://www.lyngsat.com/logo/tv/pp/presence-tv-channel-et.png
-PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png
+PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://www.presstv.ir/custom/images/site/LOGO.gif
```
Comment (LaneSh4d0w) [inline data/channels.csv:18639]:
```suggestion
PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://french.presstv.ir/img/logo.png
```
No need for two separate logo links when they're the same.

Comment (freearhey) [reply data/channels.csv:18639]: But what was wrong with the old logo?

https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png

Comment (LaneSh4d0w) [reply data/channels.csv:18639]: Ah, wrong one I've chosen.

File: data/channels.csv [hunk 2]
```diff
@@ -18636,7 +18636,8 @@ PremiumStories.it,Premium Stories,,,,IT,,,c/IT,ita,,FALSE,2015-06-23,2022-01-10,
 PremiumStoriesHD.it,Premium Stories HD,,,,IT,,,c/IT,ita,,FALSE,2018-06-04,2022-01-10,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PremiumStoriesPlus24.it,Premium Stories +24,,,,IT,,,c/IT,ita,,FALSE,2016-03-01,2017-09-04,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PresenceTVChannel.et,Presence TV Channel,,,,ET,,,c/ET,amh,,FALSE,,,,,https://www.lyngsat.com/logo/tv/pp/presence-tv-channel-et.png
-PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png
+PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://www.presstv.ir/custom/images/site/LOGO.gif
+PressTVFrench.ir,Press TV French,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,fra,news,FALSE,2007-07-08,,,https://french.presstv.ir/,https://french.presstv.ir/img/logo.png
```
Comment (LaneSh4d0w) [inline data/channels.csv:18640]:
```suggestion
PressTVFrench.ir,Press TV French,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,fra,news,FALSE,2007-07-08,,,https://french.presstv.ir/,https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png

```

**Review by freearhey — APPROVED** [review]

**Pull request merged**